concrete/config/app.php

```diff
@@ -141,6 +141,8 @@
         // Validator
         'core_validator' => '\Concrete\Core\Validator\ValidatorServiceProvider',
         'core_validator_password' => '\Concrete\Core\Validator\PasswordValidatorServiceProvider',
+        'core_validator_user_name' => '\Concrete\Core\Validator\UserNameValidatorServiceProvider',
```

Comment:
We should've had better naming for older validators. IMO `\Concrete\Core\Validator\UserName\Provider` is more descriptive and less repetitive. Just thoughts moving forward, I know reconfiguring this now isn't a good idea

Reply:
Yes, too late IMHO, I'd prefer to keep consistency. Furthermore I prefer longer class names, so that we don't have to use `as` in `use` statements in case we work with multiple providers.

Reply:
> Furthermore I prefer longer class names, so that we don't have to use as in use statements in case we work with multiple providers.

Nothing wrong with `use .. as ...`, it's there for a reason

Reply:
In any case it's too late, I'd prefer to have consistent class names.

Reply:
Yep can't change now. Just worth noting for future classes. If the namespace has all the context, there's little reason to put /more/ context in the class name